docs/source/ledger-integration-kit/index.rst

```diff
@@ -0,0 +1,32 @@
+.. Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+.. _ledger-integration-kit:
+
+DAML Ledger Integration Kit
```

Comment:
I would say this could be lowercase "kit", but it's up to you, as it's the name of your tool.

```suggestion
DAML Ledger Integration kit
```